.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
was this IDE config file checked in intentionally?

Reply:
Yes. That is automatically added by project-keeper to unify formatting accross IDEs. VSC does not read the Eclipse config (unlike IntelliJ).